Add missing pp, vpp and nodal kernels docs

GiudGiud opened the pull request:
Not ready for review. Trying to run the test suite to see if I can re-name the Volume Histogram VPP and get away without a phantom header or more work

GiudGiud committed:
Add missing documentation for some nodal scalar kernels, refs #18333 #17566 #16248 #15968

GiudGiud committed:
Add missing documentation for some postprocessors, refs #18333 #17566 #16248 #15968

GiudGiud committed:
Add missing documentation for some vector postprocessors, refs #18333 #17566 #16248 #15968

GiudGiud committed:
Rename volumehistorgram vpp to variable value volume histogram, for searchability
Fix links in doc

moosebuild commented:
Job [Documentation](https://civet.inl.gov/job/869744/) on 0fdf111 wanted to post the following:

View the site [here](https://mooseframework.inl.gov/docs/PRs/19269/site)

This comment will be updated on new commits.

moosebuild commented:
Job [Coverage](https://civet.inl.gov/job/869771/) on 0fdf111 wanted to post the following:


## Framework coverage

<table>
  <thead>
    <tr>
      <th></th>
      <th>7607a3</th>
      <th colspan=4>#19269 0fdf11</th>
    </tr>
    <tr>
      <th></th>
      <th>Total</th>
      <th>Total</th>
      <th>+/-</th>
      <th>New</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><b>Rate</b></td>
      <td>81.83%</td>
      <td>81.83%</td>
      <td>+0.00%</td>
      <td>97.73%</td>
    </tr>
    <tr>
      <td><b>Hits</b></td>
      <td>71503</td>
      <td>71505</td>
      <td>+2</td>
      <td>43</td>
    </tr>
    <tr>
      <td><b>Misses</b></td>
      <td>15875</td>
      <td>15874</td>
      <td>-1</td>
      <td>1</td>
    </tr>
  </tbody>
</table>

[Diff coverage report](https://mooseframework.inl.gov/docs/PRs/19269/coverage_diff/framework.html)

[Full coverage report](https://mooseframework.inl.gov/docs/PRs/19269/coverage/framework)


## Modules coverage

Coverage did not change

## Full coverage reports

<details>
<summary>Reports</summary>

- [`framework`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/framework)
- [`chemical_reactions`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/chemical_reactions)
- [`combined`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/combined)
- [`contact`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/contact)
- [`external_petsc_solver`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/external_petsc_solver)
- [`fluid_properties`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/fluid_properties)
- [`fsi`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/fsi)
- [`functional_expansion_tools`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/functional_expansion_tools)
- [`geochemistry`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/geochemistry)
- [`heat_conduction`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/heat_conduction)
- [`level_set`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/level_set)
- [`misc`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/misc)
- [`navier_stokes`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/navier_stokes)
- [`peridynamics`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/peridynamics)
- [`phase_field`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/phase_field)
- [`porous_flow`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/porous_flow)
- [`ray_tracing`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/ray_tracing)
- [`rdg`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/rdg)
- [`reactor`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/reactor)
- [`richards`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/richards)
- [`stochastic_tools`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/stochastic_tools)
- [`tensor_mechanics`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/tensor_mechanics)
- [`xfem`](https://mooseframework.inl.gov/docs/PRs/19269/coverage/xfem)

</details>



This comment will be updated on new commits.

GiudGiud committed:
Add missing documentation for vpps, #18333 #17566 #16248 #15968

GiudGiud committed:
Use solve=false in a test that only looks at mesh internsections

GiudGiud committed:
Add test for sampling variables along an internal sideset, refs #3087

GiudGiud committed:
Fix ups on variable doc files after looking at website (formulas, links etc)

Finish up two doc files
columns in volume histogrm of variable values
add link to quadrature in L2 difference

allow_renumbering=false not true to keep ids the same

GiudGiud commented:
if it passes, should be good to go
